traveler-ios-ui/TravelerKitUI/ViewControllers/ProductDetailViewController.swift

```diff
@@ -12,7 +12,7 @@ import TravelerKit
 class ProductDetailViewController: UIViewController {
     var product: Product?
 
-    private var productDetails: CatalogItemDetails?
+    private var bookingProductDetails: BookingItemDetails?
```

Comment:
since catalogItemDetails is still a thing you can use it here and cast it to the appropriate concrete type in your prepareForSegue pattern